packages/xod-client/src/user/actions.js

```diff
@@ -0,0 +1,10 @@
+import { UPDATE_COMPILE_LIMIT } from './actionTypes';
+
+export const updateCompileLimit = COMPILE_HOSTNAME => () => dispatch =>
+  fetch(`${COMPILE_HOSTNAME}/limits`)
```

Comment:
And I'm not sure about this URL composing.

I'd rather use an argument as it passed...
And in the container — don't pass just a constant, but call `getCompileLimitsUrl` from `siteLinks.js` (probably renamed into `urls.js`).